user/g4music/template.py

```diff
@@ -0,0 +1,29 @@
+pkgname = "g4music"
+pkgver = "3.8.2"
+pkgrel = 0
+build_style = "meson"
+# Only checks the validity of the desktop files
+configure_args = ["-Dunit_tests=disabled"]
+hostmakedepends = [
+    "meson",
+    "pkgconf"
+]
+makedepends = [
+    "desktop-file-utils",
+    "gettext-devel",
+    "gobject-introspection",
+    "gst-plugins-bad",
+    "gst-plugins-base-devel",
+    "gst-plugins-good",
+    "gst-plugins-ugly",
```

Comment:
bad/good/ugly here are not makedepends or checked afaict, at best they're regular runtime depends

but it's fine to leave them out and have them be manually installed later